### ISSUE TITLE
Ghost graphics in OpenGL mode?

### ISSUE THREAD

**Issue by TheGameratorT** [Arisotura/melonDS#452]
So, there seems to be an extra blue flower when rendering in OpenGL mode. My ROM hack has an issue that makes it disappear in the original DS render but in melonDS OpenGL it is there somehow. If I switch to software renderer it is gone.

OpenGL:
![image](https://user-images.githubusercontent.com/20758065/59563330-6d69f000-9030-11e9-958b-fa3d2efd464e.png)

Software (DS):
![image](https://user-images.githubusercontent.com/20758065/59563343-a904ba00-9030-11e9-9159-d5d9ced2a840.png)

**Comment by Arisotura** [comment]
'd have to see what is done to hide that flower

also, for the framerate limiter, your driver is forcing vsync.

**Comment by TheGameratorT** [comment]
The flower seem to disappear after I run the following:
`vramSetBankC(VRAM_C_SUB_BG_0x06200000);`

**Comment by Arisotura** [comment]
so, if I understand correctly, this works by unmapping a texture VRAM bank, such that it will be reading zeroes instead of the flower texture

which breaks here because the current OpenGL texture handling code is a bit stupid about this

**Comment by TheGameratorT** [comment]
Yeah, because when I change to Software and then back to OpenGL it is gone.

**Comment by Arisotura** [comment]
tbh this is a bit of a hack, but pointing at an issue in my GL renderer
